components/dashboard/donation/removeDonationModalBody.js

```diff
@@ -69,6 +73,7 @@ const RemoveDonation = ({ updateDonorStatus, onClose }) => {
         </FBButton>
         <FBButton
           onClick={handleDonorDowngrade}
+          isLoading={isDeleting}
```

Comment:
We need `loadingText` as well. `isLoading` has no effect without that prop. So something like `loadingText='Deleting donating…'`.

Reply:
Is loading shows the spinner for a button and makes it un clickable, that’s native chakra behavior but I’ll add loading text too!

Reply:
Oh yeah, `loadingText` is a Chakra thing too. :) Just shows the text that should be displayed during the loading state:
https://chakra-ui.com/button#button-loading-state